diff -Naur linux-4.12.5/arch/arm/Makefile linux-4.12.5-odroid/arch/arm/Makefile
--- linux-4.12.5/arch/arm/Makefile	2017-08-06 19:22:44.000000000 +0300
+++ linux-4.12.5-odroid/arch/arm/Makefile	2017-08-10 13:06:22.201131300 +0300
@@ -145,6 +145,7 @@
 ifeq ($(CONFIG_ARCH_SA1100),y)
 textofs-$(CONFIG_SA1111) := 0x00208000
 endif
+textofs-$(CONFIG_ARCH_MESON) := 0x00208000
 textofs-$(CONFIG_ARCH_MSM8X60) := 0x00208000
 textofs-$(CONFIG_ARCH_MSM8960) := 0x00208000
 textofs-$(CONFIG_ARCH_AXXIA) := 0x00308000
